codespell_lib/data/dictionary.txt

```diff
@@ -4333,51 +4361,110 @@ archine->archive
 archines->archives
 architct->architect
 architcts->architects
+architctural->architectural
+architcturally->architecturally
 architcture->architecture
 architctures->architectures
+architec->architect
 architecht->architect
 architechts->architects
+architechtural->architectural
 architechturally->architecturally
 architechture->architecture
 architechtures->architectures
+architecs->architects
 architectual->architectural
+architectually->architecturally
+architectue->architecture
+architectues->architectures
 architectur->architecture
 architecturs->architectures
 architecturse->architectures
+architecural->architectural
+architecurally->architecturally
 architecure->architecture
 architecures->architectures
+architecut->architect
+architecutral->architectural
+architecutrally->architecturally
 architecutre->architecture
 architecutres->architectures
+architecuts->architects
+architecutural->architectural
+architecuturally->architecturally
 architecuture->architecture
 architecutures->architectures
+architet->architect
+architetc->architect
+architetcs->architects
+architetcural->architectural
+architetcurally->architecturally
 architetcure->architecture
 architetcures->architectures
+architets->architects
+architetural->architectural
+architeturally->architecturally
 architeture->architecture
 architetures->architectures
+architural->architectural
+architurally->architecturally
 architure->architecture
 architures->architectures
 archiv->archive
+archivd->archived
 archivel->archival
+archivr->archiver
+archivrs->archivers
+archivs->archives
 archor->anchor
+archtect->architect
+archtects->architects
+archtectural->architectural
+archtecturally->architecturally
 archtecture->architecture
 archtectures->architectures
+archtiect->architect
+archtiects->architects
+archtiectural->architectural
+archtiecturally->architecturally
 archtiecture->architecture
 archtiectures->architectures
+archtitect->architect
+archtitects->architects
+archtitectural->architectural
+archtitecturally->architecturally
 archtitecture->architecture
 archtitectures->architectures
 archtype->archetype
 archtypes->archetypes
+archve->archive
```

Comment:
Perhaps add the `archival` adjective? 
⚠ In some cases, relevant typos could be from `archrival`.

Reply:
Done. I didn't see any typos from `archrival`.